packages/calcite-components/src/components/shell-panel/shell-panel.scss

```
@@ -10,6 +10,10 @@
  * @prop --calcite-shell-panel-max-height: When `layout` is `horizontal`, or `layout` is `vertical` and `displayMode` is `float-content` or `float`, specifies the maximum height of the component.
  * @prop --calcite-shell-panel-min-height: When `layout` is `horizontal`, or `layout` is `vertical` and `displayMode` is `float-content` or `float`, specifies the minimum height of the component.
  * @prop --calcite-shell-panel-z-index: Specifies the z-index value for the component.
+ * @prop --calcite-shell-panel-border-color: When `displayMode` is `"float-all"`, specifies the border color of the component.
+ * @prop --calcite-shell-panel-corner-radius: When `displayMode` is `"float-all"`, specifies the corner radius of the component.
+ * @prop --calcite-shell-panel-shadow: When `displayMode` is `"float-all"`, specifies the shadow of the component.
```

nitpick for doc consistency 📚:

```suggestion
 * @prop --calcite-shell-panel-border-color: When `displayMode` is `"float-all"`, specifies the component's border color.
 * @prop --calcite-shell-panel-corner-radius: When `displayMode` is `"float-all"`, specifies the component's corner radius.
 * @prop --calcite-shell-panel-shadow: When `displayMode` is `"float-all"`, specifies the component's shadow.
```

---

Updated 👍